src/libs/actions/Report.js

```diff
@@ -733,12 +733,11 @@ function createOptimisticReport(participantList) {
 }
 
 /**
- * @param {Number} reportID
  * @param {String} [text]
  * @param {File} [file]
  * @returns {Object}
  */
-function buildOptimisticReportAction(reportID, text, file) {
+function buildOptimisticReportAction(text, file) {
```

Comment:
I'm confused about why we are removing the `reportID` from here.

Reply:
Because it wasn't being used by the method and vscode was graying it out : )

Reply:
Ah ok I probably missed it when making [these changes](https://github.com/Expensify/App/pull/9556/files) thanks!